setup.py

```diff
@@ -73,10 +73,10 @@
     },
     entry_points={
         'console_scripts': [
-            'pix2tex_gui = pix2tex.gui:main',
-            'pix2tex_cli = pix2tex.cli:main',
-            'latexocr = pix2tex.gui:main',
-            'pix2tex = pix2tex.cli:main',
+            'pix2tex_gui = pix2tex.__main__:main',
+            'pix2tex_cli = pix2tex.__main__:main',
+            'latexocr = pix2tex.__main__:main',
+            'pix2tex = pix2tex.__main__:main',
```

Comment:
This does not seem right. Would it not work unchanged?

Reply:
I change `pix2tex/{cli,gui}.py`'s `main()`:

```python
def main(arguments):
    with in_model_path():
        app = QApplication(sys.argv)
        ex = App(arguments)
        sys.exit(app.exec_())
```

Now `pix2tex/__main__.py`'s `main()` parse arguments then pass the arguments to them:

```python
name = os.path.split(sys.argv[0])[-1]
if arguments.gui or arguments.gnome or name in ['pix2tex_gui', 'latexocr']:
    from .gui import main
else:
    from .cli import main
main(arguments)
```

Because I notice cli and gui have many same arguments in their parsers. Writing once is more convenient than copying twice.

And I have tested in `pip install -e .`.